sim/dex-data.ts

```diff
@@ -317,7 +317,7 @@ export class DexTypes {
 	}
 
 	isName(name: string): boolean {
```

Comment:
Shit I forgot to change this to `string | null | undefined`

Reply:
solved in https://github.com/smogon/pokemon-showdown/pull/11172